File: 240423/007/007.cpp
#include <iostream>

using namespace std;

class Spy {
public:
	string secretcode; char meetingplace; int time;

	Spy(string s, char m, int t) {
		this->secretcode = s;
		this->meetingplace = m;
		this->time = t;
	}
};

int main() {
	string s; char m; int t;
	cin >> s >> m >> t;
	Spy Spy1(s, m, t);
	cout << "secret code : " << Spy1.secretcode << '\n' 
		 << "meeting point : " << Spy1.meetingplace << '\n'
		 << "time : " << Spy1.time;
	return 0;
}

File: 240501/007/007.py
class Spy:
    def __init__(self, secretcode, meetingpoint, time):
        self.s = secretcode
        self.m = meetingpoint
        self.t = time

secretcode, meetingpoint, time = "codetree", "L", 13

spy1 = Spy(secretcode, meetingpoint, time)
print("secret code :", spy1.s)
print("meeting point :", spy1.m)
print("time :", spy1.t)